src/applications/simple-forms/21-0966/containers/ConfirmationPage.jsx

```diff
@@ -21,170 +21,159 @@ import {
 } from '../config/helpers';
 import { benefitPhrases, veteranBenefits } from '../definitions/constants';
 
-export class ConfirmationPage extends React.Component {
-  componentDidMount() {
-    focusElement('h2');
+export const ConfirmationPage = props => {
+  useLayoutEffect(() => {
+    focusElement('h2', null, 'va-alert');
```

Comment:
by calling focus element in this way, it ensures that it waits for the web component to asynchronously paint first